fw-child/resources/scss/_vendor.scss

```diff
@@ -27,23 +27,33 @@
   flex-wrap: wrap;
   list-style: none;
   padding: 0;
+  margin-bottom: 0;
 
   li {
-    display: flex;
-    margin-bottom: 0.5rem;
-
-    &:not(.share-permalink-wrap) {
-      order: 2;
-
-      .label {
-        display: none;
-      }
+    &:not(:first-child):not(.share-permalink-wrap) {
+      margin-left: 0.75rem;
```

Comment:
I know that there will be updates in the Builder to add other responsive styling, and that the social links would maybe never end up being displayed one under the other, but maybe the `<li>` should have a _right_-margin instead of a _left_-margin, to prevent this possible misalignment if they fall on more than one line:

![image](https://github.com/CanadianClimateDataPortal/climatedata-wp-theme/assets/159493515/ebab4e2e-c529-458a-a34d-3223f09ab9b0)

Reply:
@xfrenette-crim this has been fixed. We also added some bottom margin to not have condensed view.